react/features/calendar-sync/functions.web.ts

```diff
@@ -80,13 +84,13 @@ export function _fetchCalendarEntries(
         })
         .then(() => dispatch(integration.getCalendarEntries(
             FETCH_START_DAYS, FETCH_END_DAYS)))
-        .then(events => _updateCalendarEntries.call({
+        .then((events: Object[]) => _updateCalendarEntries.call({
```

Comment:
What's preferable, `Object[]` or `any[]` ?

Reply:
In this case `Object[]` is fine

Reply:
Generally speaking, `any` will be more useful because unless you care only calling methods in the `Object` prototype, you'll have TS errors.